### PR TITLE
[MSK-2020] add DOD moscow-2020

### PR DESCRIPTION
**Pull request by Frodox**
Added Moscow 2020 event with best hopes :)

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Hi, @frodox - we have changed the code of conduct template. Please use this one: https://raw.githubusercontent.com/devopsdays/devopsdays-web/master/utilities/examples/content/events/yyyy-city/conduct.md

File: data/events/2020-moscow.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+  - name: Andrey Levkin
+    bio: Lead System Engineer
+    employer: EPAM
+    facebook: https://www.facebook.com/levkin
+    linkedin: https://www.linkedin.com/in/alevkin/
+    image: andrey.jpg
+
+
+organizer_email: "organizers-moscow-2020@devopsdays.org"
```
Comment (bridgetkromhout) [inline data/events/2020-moscow.yml:107]: We no longer use this email format - we use moscow@devopsdays.org

File: data/events/2020-moscow.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+proposal_email: "info@devopsdays.ru"
```
Comment (bridgetkromhout) [inline data/events/2020-moscow.yml:108]: We no longer use this field.

**Comment by mattstratton** [comment]
@Frodox can you review the notes from @bridgetkromhout and make the needed changes? thank you!

**Comment by Frodox** [comment]
@mattstratton @bridgetkromhout 

Thank you for review, updated! :)

**Comment by bridgetkromhout** [comment]
Thanks, @Frodox. Since you force-pushed, it's much more time consuming to review the changes. I can't work on it this morning; I'll try to get to it tonight (US timezone). In the future, please do not use force-push in this repo. Thanks!

**Comment by Frodox** [comment]
Sorry :( 
Thinking about clean git-history...

**Comment by mattstratton** [comment]
>Thinking about clean git-history...

we usually do "squash and merge" when merging PRs so it ends up being one clean commit in the main repo history, so don't worry about having lots of commits in the PR!

**Pull request merged**